### ISSUE TITLE
[Bug] jQuery UI Datepicker dependecies not localized

### ISSUE THREAD

**Issue by EdenK** [pojome/elementor#5192]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

When using datepicker in a widget it is not getting localized.

Wordpress core information:
https://make.wordpress.org/core/2016/07/06/jquery-ui-datepicker-localization-in-4-6/

## Steps to reproduce
```
class My_Custom_Widget extends Widget_Base {
	...

	public function get_script_depends() {
		return [ 'jquery-ui-datepicker' ];
	}

	...
}
```

Wordpress in different locale like he_IL or fr_FR

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

**Comment by bainternet** [comment]
Thanks for reporting,
Please follow guidelines for opening a new issue report, so we can help you better.
This report is missing key information: 

* Prerequisites.
* Steps to replicate.
* Isolating the problem.
* System info.

anyway, it seems to work just fine, can you check the source code of the editor and see if you have something like this: `jQuery.datepicker.setDefaults`?

**Comment by EdenK** [comment]
I don't have it.

The problem is in the frontend you run the function `enqueue_scripts` in the `wp_footer` so the hook
`add_action( 'wp_enqueue_scripts', 'wp_localize_jquery_ui_datepicker', 1000 );`

Is not working.

Iv'e have to add:
`add_action( 'elementor/frontend/after_enqueue_scripts', 'wp_localize_jquery_ui_datepicker', 1000 );`

In order to add the hook to the `enqueue_scripts`.

**Comment by bainternet** [comment]
@EdenK 

I see what you mean, well yeah, Elementor enqueues the scripts only once it knows it is needed meaning after processing the widgets in the page and that is why it is enqueued in the footer.

And I guess I can understand why they hooked it only once and not in the footer as well.
So your solution is a good way to go.

Glad you figured it out.
